packages/wallet/src/SingleAddressWallet.ts

```diff
@@ -233,5 +233,22 @@ export class SingleAddressWallet implements Wallet {
     this.protocolParameters$.complete();
     this.genesisParameters$.complete();
     this.#tip$.complete();
+    this.addresses$.complete();
+  }
+
+  #initializeAddress(existingAddress?: GroupedAddress): Observable<GroupedAddress[]> {
```

Comment:
This is good for a single-address wallet but doesn't align with multi-address wallet semantics. I think it's fine to just refactor later given it's a private method though